libraries/provider_hab_package.rb

```diff
@@ -110,9 +110,7 @@ def strip_version(name)
         def platform_target
           if platform_family?('windows')
             'target=x86_64-windows'
-          elsif platform_family?('rhel') && node['platform_version'].to_i < 6
-            'target=x86_64-linux-kernel2'
-          elsif platform_family?('suse') && node['platform_version'].to_i < 6
+          elsif node['kernel']['release'].to_f < 3.0
```

Comment:
You could easily just .to_i < 3 here as well. No need to treat it as a float

Reply:
You're right. It think at first I was accidentally calling version rather than release and didnt understand why I was failing and forgot to change it when i caught my original mistake. Will change that up and push it again